### PR TITLE
Rework backend to return 2 payloads

### PR DESCRIPTION
**Pull request by Yelhedhli**

### CODE REVIEW

File: datahub/controllers/datahub.go [hunk 1]
```diff
@@ -55,17 +55,28 @@ func (c ClientPayload) fillDefaults() {
...
 }
 
 type ResultsDataPayload struct {
 	SearchResult []maps.PlacesSearchResult
 }
 
+type SessionStateDataPayload struct {
+	NumStartRating       int
+	NumUpdateRestaurants int
+	NumFinishRating      int
```
Comment (eriklungulescu) [inline datahub/controllers/datahub.go:77]: Need num in session

File: datahub/controllers/datahub.go [hunk 2]
```diff
@@ -103,30 +114,40 @@ func (h DataHubController) handleSession() { //sub to channel, continuously re p
...
+
+				err = h.redisClient.Publish(ctx, "datahub"+h.sessionID.String(), marshaledPayload).Err()
+				if err != nil {
+					panic(err)
+				}
+
+			}
+		}
+	}
+}
```
Comment (eriklungulescu) [inline datahub/controllers/datahub.go:153]: lgtm

**Review by eriklungulescu — APPROVED** [review]
lgtm

File: datahub/controllers/datahub.go (hunk 1 above)
Comment (Yelhedhli) [reply datahub/controllers/datahub.go:78]: done

**Pull request merged**